pandas/tests/indexing/test_loc.py

```diff
@@ -1130,3 +1130,16 @@ def test_loc_with_period_index_indexer():
     tm.assert_frame_equal(df, df.loc[list(idx)])
     tm.assert_frame_equal(df.iloc[0:5], df.loc[idx[0:5]])
     tm.assert_frame_equal(df, df.loc[list(idx)])
+
+def test_loc_replace_subset_with_another():
```

Comment:
I'd call it `test_loc_replace_subset_with_subset`

Reply:
Done